app/models/payout.rb

```diff
@@ -39,6 +39,10 @@ class Payout < ApplicationRecord
 	scope :pending, -> {where(status: 'pending')}
 	scope :paid,    -> {where(status: ['paid', 'succeeded'])}
 
+	def create
+		self.publish_created
+	end
+
```

Comment:
I'm not sure I understand this method, how were you intending on using this method?

Reply:
Was trying to call `publish_created` from in InsertPayout, but the only representation of a payout we get there (I think) is a hash. So I put the call into the payout create method to avoid that issue. Open to other workarounds/suggestions tho

Reply:
You do get a hash in InsertPayout but it should have the Payout id in it. You can use `Payout#find` to get the actual object.